github.com.hashicorp.go.kms.wrapping.v2.types.proto

```diff
@@ -97,8 +103,18 @@ message Options {
 
 	// Wrapper-specific configuration to pass along
 	map<string, string> with_config_map = 30;
+
+	// The purposes of the key being specified
+	repeated KeyPurpose with_key_purposes = 40;
+
+	// The type of the key being specified
+	KeyType with_key_type = 50;
+
+	// optional bytes of entropy
```

Comment:
Two things here:

1) The rest of the comments start with a capital letter, this does not
2) Should it be `with_random_bytes`? I'm fine with "no", just raising it as within the Go world "random" can mean a function from big/int or crypto/rand or can mean a reader (e.g. rand.Reader)...might be worth being explicit. Feel free to ignore.

Reply:
I agree with the suggestion. ty!